samples/cmake/ozz-animation/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+if(NXDK)
+  # Hack for memory.h
+  file(TOUCH ${CMAKE_CURRENT_BINARY_DIR}/memory.h)
+  include_directories(SYSTEM "${CMAKE_CURRENT_BINARY_DIR}/")
```

Comment:
I'm not sure what `<memory.h>` is, but an empty file is enough.

Fix upstream in ozz-animation by removing the inclusion if it's really not necessary.
Fix in nxdk by adding a dummy file (not sure where? winapi?) if it's actually used in ozz-animation.